Cargo.toml

```diff
@@ -7,11 +7,17 @@ edition = "2018"
 [dependencies]
 anyhow = "1"
 async-trait = "0.1"
+base64 = "0.12.3"
 bitcoin = { version = "0.23.0", features = ["rand", "use-serde"] }
+bitcoin-harness = { git = "https://github.com/coblox/bitcoin-harness-rs.git", rev = "f4f294863f2f65d5379ccd5f30896c15bb3cf3ca" }
```

Comment:
This is what I wanted to prevent :( This dependency should only live in `[dev-dependencies]`